### PR TITLE
Translation and text display fixes

### PR DESCRIPTION
**Pull request by kamieniarz**
Translation and text display fixes

## Description
Simply added some more translations and fixed text display by removing username in tracked list notification and switching places in invasion info

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/lib/discord/commando/commands/tracked.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ exports.run = async (client, msg, command) => {
 
 		try {
 			const hastelink = await client.hastebin(message)
-			return await msg.reply(`${target.name} ${client.translator.translate('tracking list is quite long. Have a look at')} ${hastelink}`)
+			return await msg.reply(`${client.translator.translate('tracking list is quite long. Have a look at')} ${hastelink}`)
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:143]: Why did you remove the users name here?

**Comment by Petap0w** [comment]
Great addition, although if doing that then you should translate all in a reworked line (like level, distance, etc.) and leave the `:` out of the translated text?

File: src/lib/discord/commando/commands/tracked.js (hunk 1 above)
Comment (kamieniarz) [reply src/lib/discord/commando/commands/tracked.js:143]: Basically when using `!tracked` it's you that's asking for tracked list so why to list your name there? It just doesn't look right

**Comment by kamieniarz** [comment]
@Petap0w done

File: src/lib/discord/commando/commands/tracked.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ exports.run = async (client, msg, command) => {
...
+				message = message.concat(`\n**${client.translator.translate(`${monsterName}`)}** ${client.translator.translate(`${formName}`)} ${monster.distance ? ` | distance: ${monster.distance}m` : ''} | iv: ${miniv}%-${monster.max_iv}% | ${client.translator.translate('cp')}: ${monster.min_cp}-${monster.max_cp} | level: ${monster.min_level}-${monster.max_level} | ${client.translator.translate('stats')}: ${monster.atk}/${monster.def}/${monster.sta} - ${monster.max_atk}/${monster.max_def}/${monster.max_sta} | ${client.translator.translate('greatpvp')}: ${greatLeague} | ${client.translator.translate('ultrapvp')}: ${ultraLeague}${monster.gender ? ` | ${client.translator.translate('gender')}: ${client.utilData.genders[monster.gender].emoji}` : ''}`)
```
Comment (Petap0w) [inline src/lib/discord/commando/commands/tracked.js:72]: there we could add translation for distance, iv and level

File: src/lib/discord/commando/commands/tracked.js [hunk 3]
```diff
@@ -119,18 +119,18 @@ exports.run = async (client, msg, command) => {
 				let genderText = ''
 				let typeText = ''
 				if (!invasion.gender || invasion.gender === '') {
-					genderText = 'Gender: any, '
+					genderText = client.translator.translate('any')
```
Comment (jfberry) [inline src/lib/discord/commando/commands/tracked.js:122]: I think petap0w meant do `${client.translator.translate('Gender'):${client.translator.translate('any')`

File: src/lib/discord/commando/commands/tracked.js [hunk 4]
```diff
@@ -119,18 +119,18 @@ exports.run = async (client, msg, command) => {
...
 				} else if (invasion.gender === 1) {
-					genderText = 'Gender: male, '
+					genderText = client.translator.translate('male')
 				} else if (invasion.gender === 2) {
-					genderText = 'Gender: female, '
+					genderText = client.translator.translate('female')
 				}
 				if (!invasion.grunt_type || invasion.grunt_type === '') {
-					typeText = 'Any'
+					typeText = client.translator.translate('Any')
```
Comment (Petap0w) [inline src/lib/discord/commando/commands/tracked.js:129]: we could swap to lowercase to be consistent with other files using "any"

File: src/lib/discord/commando/commands/tracked.js (hunk 3 above)
Comment (Petap0w) [reply src/lib/discord/commando/commands/tracked.js:122]: no it's ok the way he did it with gender in next line

Comment (kamieniarz) [reply src/lib/discord/commando/commands/tracked.js:122]: I think my solution is better than that. Tested, works

File: src/lib/discord/commando/commands/tracked.js [hunk 5]
```diff
@@ -119,18 +119,18 @@ exports.run = async (client, msg, command) => {
...
 				} else {
 					typeText = invasion.grunt_type
 				}
-				message = message.concat(client.translator.translate(`\nInvasion: ${genderText}Grunt type: ${typeText}`))
+				message = message.concat(`\n${client.translator.translate('Invasion')}: ${client.translator.translate('Grunt type')}: ${typeText}, ${client.translator.translate('Gender')}: ${genderText}`)
```
Comment (Petap0w) [inline src/lib/discord/commando/commands/tracked.js:133]: and we already have `invasion` lower case from command translation as well

File: src/lib/discord/commando/commands/tracked.js (hunk 1 above)
Comment (Petap0w) [reply src/lib/discord/commando/commands/tracked.js:143]: I liked having the name there, made it look like a conversation - I wouldn't remove it. If removed then `Tracking list...` needs upper case for T

File: src/lib/discord/commando/commands/tracked.js (hunk 2 above)
Comment (kamieniarz) [reply src/lib/discord/commando/commands/tracked.js]: Done

File: src/lib/discord/commando/commands/tracked.js (hunk 4 above)
Comment (kamieniarz) [reply src/lib/discord/commando/commands/tracked.js]: Done

File: src/lib/discord/commando/commands/tracked.js (hunk 1 above)
Comment (kamieniarz) [reply src/lib/discord/commando/commands/tracked.js:143]: It doesn't look like conversation at all since there are already lines about notifications enabled, areas and location before thet part. I'll change that `t` then. Anyway I don't mind if you undo my changes at that part

**Comment by Petap0w** [comment]
Ready for merge
